modules/react/combobox/lib/ComboboxMenu.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import {createSubcomponent} from '@workday/canvas-kit-react/common';
+import {Menu} from '@workday/canvas-kit-react/menu';
+
+import {useComboboxModel} from './useComboboxModel';
+import {ComboboxMenuList} from './ComboboxMenuList';
+import {ComboboxMenuItem} from './ComboboxMenuItem';
+import {ComboboxCard} from './ComboboxCard';
+
+export interface ComboboxMenuProps {
+  /**
+   * Children of the ComboboxMenu.
+   */
+  children: React.ReactNode;
+}
+
+export const ComboboxMenu = createSubcomponent()({
+  modelHook: useComboboxModel,
+  subComponents: {
+    /**
+     * The "Popper" of a combobox menu. The popper will appear around the
+     * {@link ComboboxInput Combobox.Input}. It renders a `div` element that is portalled to the
+     * `document.body` which is controlled by the {@link PopupStack}. The `PopupStack` is not part
+     * of React. This means no extra props given to this component will be forwarded to the `div`
+     * element, but the `ref` will be forwarded.
+     */
+    Popper: Menu.Popper,
+    /**
+     * The combobox menu list follows the Collections API. A list can either contain static items or
+     * a render prop and `items` to the model.
+     *
+     * ```tsx
+     * const MyComponent = () => {
+     *   const model = useComboboxModel({
+     *     items: ['First Item', 'Second Item']
+     *   })
+     *
+     *   return (
+     *     <Combobox model={model}>
+     *       // other combobox subcomponents
+     *       <Combobox.List>
+     *         {(item) => <Combobox.Item>{item}</Combobox.Item>}
+     *       </Combobox.List>
+     *     </Combobox>
```

Comment:
```suggestion
     *     <Combobox model={model}>
     *       // other combobox subcomponents
     *       <Combobox.Menu.List>
     *         {(item) => <Combobox.Menu.Item>{item}</Combobox.Menu.Item>}
     *       </Combobox.Menu.List>
     *     </Combobox>
```

Not sure if we need to wrap the `Combobox.Menu.List` in a `Combobox.Menu.Popper` and `Combobox.Menu.Card` as well or if that's what "other combobox subcomponents" is meant to represent.